src/BIP44CoinTypeNode.ts

```diff
@@ -75,19 +76,28 @@ export class BIP44CoinTypeNode implements BIP44CoinTypeNodeInterface {
    * @param json - The {@link JsonBIP44Node} for the key of this node.
    * @param coin_type - The coin_type index of this node. Must be a non-negative
    * integer.
+   * @param cryptographicFunctions - The cryptographic functions to use. If
+   * provided, these will be used instead of the built-in implementations.
    */
-  static async fromJSON(json: JsonBIP44Node, coin_type: number) {
+  static async fromJSON(
+    json: JsonBIP44Node,
+    coin_type: number,
+    cryptographicFunctions?: CryptographicFunctions,
```

Comment:
Let's add similar tests to `SLIP10Node` that the functions are passed down correctly